EvalView/views.py

```diff
@@ -244,7 +250,8 @@ def direct_assessment(request, code=None, campaign_name=None):
             '(middle) or preference for <em>Candidate B</em> (right).'
         )
 
-    campaign_opts = campaign.campaignOptions or ""
+    campaign_opts = set((campaign.campaignOptions or "").lower().split(";"))
```

Comment:
I think this changes the assumption that we had (undocumented, but in use): `in` operator on a string allowed for some variations, e.g. `namedentit` could be triggered by both `NamedEntityEval;foo;bar` or `NamedEntities`. 

Hm, this may be the reason why few regression tests fail.